src/witness/yamlWitness.ml

```diff
@@ -340,11 +328,11 @@ struct
         entries
     in
 
-    (* Generate precondition invariants.
+    (* Generate precondition loop invariants.
```

Comment:
@jerhard How does this change affect our planned usage for precondition invariants for the project with LMU? We need general precondition invariants there iirc?

Reply:
Having `precondition_location_invariant` entries is just a matter of defining the corresponding types and doing what we already have for `location_invariant` vs `loop_invariant`: use a different predicate for filtering nodes.